docs/i18n/id/docusaurus-plugin-content-docs/current/tutorials/simple-todo-list/1-installation.md

```diff
@@ -1,34 +1,34 @@
 ---
-title: Installation
+title: Instalasi
 ---
 
-In this tutorial you will learn how to create a basic web application with FoalTS. The demo application is a simple to-do list with which users can view, create and delete their tasks.
+Pada tutorial kali ini kita akan coba membuat aplikasi web dengan Foal. Aplikasinya berupa todo-list sederhana, di mana user bisa melakukan view, create dan delete catatan to-do mereka.
 
-> **Requirements:**
+> **Yang diperlukan:**
 >
-> [Node.js](https://nodejs.org/en/) 10 or greater
+> [Node.js](https://nodejs.org/en/) versi 10 ke atas
 
-## Create a New Project
+## Memulai Proyek Baru
 
-First you need to install globaly the *Command Line Interface (CLI)* of FoalTS. It will help you create a new project and generate files all along your development.
+Pertama-tama kita perlu menginstal Foal *Command Line Interface (CLI)* secara global. Foal CLI ini akan kita gunakan untuk memulai proyek baru serta membuat file pada saat develop.
 
 ```sh
 npm install -g @foal/cli
 ```
 
-Then create a new application.
+Kemudian buat sebuah aplikasi baru.
 
 ```sh
 foal createapp my-app
 ```
 
-:::note
+:::catatan
```

Comment:
Oh I see, that's why it looks weird -- expecting a visual clue but instead a triple : appearing.
Btw, I did translate it because the Spanish version did lol.